.github/workflows/ci.yaml

```diff
@@ -0,0 +1,47 @@
+name: CI
```

Comment:
hasan added these github action workflows recently - are they showing up in this PR too because we need to rebase the work in this branch onto the latest from main?

Reply:
I'll rebase